source/scss/components/_messages.scss

```diff
@@ -0,0 +1,62 @@
+.fti-message {
+  color: $white;
+  padding: .875rem .9375rem;
+
+  &.info-message {
+    background-color: $cyan;
+  }
+
+  .message-container {
+    align-items: center;
+    display: flex;
+    min-height: 60px;
```

Comment:
I think it is good if the content defines the height, so maybe you know of a reason, why there is a min-height. Perhaps there is a way of creating distance from content to element border with a padding.